### PR TITLE
Ft/about

### PR DESCRIPTION
**Pull request by nsengaqueen**
The about section with it's responsiveness

### CODE REVIEW

File: src/sections/About.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<section class="m-8 lg:m-8 p-5 md:p-8 lg:p-32 font-outfit md:text-[2rem] lg:text-5xl">
```
Comment (l-isaro) [inline src/sections/About.html:1]: The padding seems to be too big on the mobile breakpoint. Maybe you can decrease it.

File: Images/Group 4.png
Comment (Algor-Fernand) [inline Images/Group 4.png:1]: Are we supposed to have this image in this branch, also I think we had decided on being consistent in the naming and the extension we gave our images or icons.

**Review by Algor-Fernand — CHANGES_REQUESTED** [review]
I have left a commment. You can look into it.

File: Images/Group 4.png
Comment (nsengaqueen) [reply Images/Group 4.png:1]: It was an error we deleted it!

**Review by Algor-Fernand — APPROVED** [review]

**Pull request merged**